unlock-app/src/components/interface/SignUp.tsx

```diff
@@ -148,3 +159,7 @@ const Confirmation = styled.div`
     font-weight: bold;
   }
 `
+
+const LinkButton = styled.a`
+  cursor: pointer;
```

Comment:
Is that CSS rules required? `a` should default to pointer?

Reply:
I believe that because these `a` tags don't have `href`s, they won't have a pointer.